lib/pdf_fill/extras_generator.rb

```diff
@@ -65,6 +66,23 @@ def set_font(pdf)
       pdf.font('Roboto')
     end
 
+    def set_header(pdf)
+      pdf.repeat :all do
+        pdf.bounding_box(
+          [pdf.bounds.left, pdf.bounds.top],
+          width: pdf.bounds.width
+        ) do
+          formatted_form_id = @form_id.sub(/V2\z/, '')
```

Comment:
This is neat, although I think HashConverter and ExtrasGenerator don't care about the `form_id` and shouldn't be responsible for doing this regex sub. What if you moved it up to `Filler#process_form` and just called the argument something like `form_name`?